### DISCUSSION TITLE
chart easing

### DISCUSSION THREAD

**Discussion by sergejbuneev** [klinecharts/KLineChart#270] [General]
Good day! I'm trying to set graphs with smooth drawing(easing). Could you give some advices how to do this?
Below is a short video of how I expect to set up

https://user-images.githubusercontent.com/99341029/178009534-d098e7b3-2dc4-43fd-83b6-3129ae47196c.mp4

**Comment by liihuu** [comment]
It's hard to do this.

**Reply by sergejbuneev** [reply]
hard does not meen impossible))))) maby you have some ideas how to realize this?) will be very grateful for any advises)))

**Reply by liihuu** [reply]
There is currently no way to do this, it needs support from within the chart.
